extensions/ql-vscode/src/interface.ts

```diff
@@ -235,18 +261,10 @@ export class InterfaceManager extends DisposableObject {
         this._panelLoadedCallBacks = [];
         break;
       case 'changeSort':
-        await this.changeSortState(query =>
-          query.updateSortState(
-            this.cliServer,
-            msg.resultSetName,
-            msg.sortState
-          )
-        );
+        await this.changeRawSortState(this.cliServer, msg.resultSetName, msg.sortState);
```

Comment:
Do we need to pass in `this.cliServer` since it is already a property on `this`?

Reply:
No, good catch, will just grab it directly